copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    las_filename = f"{gliht_las_file[0]}"
</pre><p>Do you need to coerce it to string like this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good point, thanks! Not necessary, I can just use the variable name here. Updated it accordingly in the revised notebook.</p>